content/sensu-go/5.5/reference/agent.md

```diff
@@ -572,14 +572,19 @@ Flags:
 | backend-url |      |
 --------------|------
 description   | ws or wss URL of the Sensu backend server. To specify multiple backends using `sensu-agent start`, use this flag multiple times.
-type          | String
+type          | 
```

Comment:
What type should we list here? String slice, list, array of strings?

Reply:
I think we should avoid using `slice`. While it's not exclusive to Go, it's kind of specific so I wouldn't expect everyone to know this term.

I like `list` because it's high-level and it doesn't imply much; `StringSlice` flags can be used in many ways so it kind of reflects that, e.g.
- `--subscriptions foo`
- `--subscriptions foo,bar`
- `--subscriptions foo --subscriptions bar`